pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.3.5.RELEASE</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+
+    <groupId>uk.gov.companieshouse</groupId>
+    <artifactId>third-party-test-harness</artifactId>
+    <version>unversioned</version>
+    <name>third-party-test-harness</name>
+    <description>Third party web application</description>
+
+    <properties>
+        <java.version>1.8</java.version>
```

Comment:
Just a comment for the future work coming, maybe we could look into using java 11 instead

Reply:
Yeah agree, i think the POC was started with 1.8 but for any future work will look to use latest version